app/uk/gov/hmrc/individualdetailsdesstub/connector/ApiPlatformTestUserConnector.scala

```diff
@@ -43,5 +44,6 @@ class ApiPlatformTestUserConnector @Inject()(servicesConfig: ServicesConfig, htt
   private def getTestIndividual(url: String)(implicit hc: HeaderCarrier) =
     http.GET[TestIndividual](url) recover {
       case _: NotFoundException => throw new TestUserNotFoundException
+      case ex: UpstreamErrorResponse if ex.statusCode == 404 => throw new TestUserNotFoundException
```

Comment:
are these two clauses not the same? can they not be merged into a single clause?